realm/realm-library/src/main/java/io/realm/internal/Group.java

```diff
@@ -226,6 +226,15 @@ public void writeToFile(File file, byte[] key) throws IOException {
         return nativeWriteToMem(nativePtr);
     }
 
+    /*
+     * Check if the Group contains any objects.
+     *
+     * @return {@code true} if empty, @{code false} otherwise.
+     */
+    public boolean isEmptyTables() {
```

Comment:
It seems `Group.size()` and `Group.isEmpty()` are useless now? Shall we just refactor them instead of creating a new method?